src/scripts/libs/language-url-parameter.ts

```diff
@@ -0,0 +1,22 @@
+import {Language} from '../types/language';
+// parses window.location and reads the story tags from query params
+//
+// note: we do not use the location.search prop here because the HashRouter
+
+// stores the query parameters in the location.hash prop
+export function parseUrl(): Language | null {
+  const {hash} = location;
+
+  const queryString = hash.substr(hash.indexOf('?'));
+  const urlParams = new URLSearchParams(queryString);
+  const languageParam = urlParams.get('lng') as Language;
+
+  if (
+    (languageParam && Object.values(Language).includes(languageParam)) ||
+    !languageParam
```

Comment:
Why is there the OR statement with `|| !languageParam`? Shouldn't ir just return null if the languageParam is not included in the list?